pkg/export/export.go

```diff
@@ -159,6 +160,8 @@ type ExporterOpts struct {
 	DisableAuth bool
 	// A user agent string added as a suffix to the regular user agent.
```

Comment:
We specifically want to prevent tampering with the user-agent as much as possible, which is why we [hardcode](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/2e01f7369ed284c77a401c63b53b145eeeec87b1/pkg/export/export.go#L230) the [`ClientName` and `Version`](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/2e01f7369ed284c77a401c63b53b145eeeec87b1/pkg/export/export.go#L472-L476) into the prefix.

However, we allow the flag as optional argument to provide any additional information that can't be inferred by the library directly.

I do understand the confusion though, so I'll rename the variable. Let me know what you think.

Reply:
As this PR updates flag interface, I suggest updating [`Version`](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/2e01f7369ed284c77a401c63b53b145eeeec87b1/pkg/export/export.go#L472-L476) in this PR to `0.6.0`, as the interface is different after this PR is merged. What do you think?

Reply:
Sure - SGTM. We can do that when we cut the release.

It also probably makes sense to bump a minor due to the target status reporting API [changes](https://github.com/GoogleCloudPlatform/prometheus-engine/commit/29d400bfdc3dbd9414e94d38e8e5ce5bff7ad892#diff-918b2f2eda7a36f6f638775958962c3aeeaa1272fa163f6f4a3633da478f667f) in PodMonitoring as well cc @TheSpiritXIII @lyanco